docker/opensource-docker/conf/core-site.xml

```diff
@@ -12,7 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+-->
```

Comment:
When I clone this repository, the licence header is already in the config and shell files. But in the config files(xml). The license header can't be parsed correctly. so I add `"<!--..-->"` to note the license header.